vagga.yaml

```diff
@@ -0,0 +1,27 @@
+minimum-vagga: v0.7.0
+containers:
+  pip-freeze:
+    setup:
+    - !Alpine v3.4
+    - !Py3Install
+      - pip
+      - 'Django >=1.9, <2.0'
+      - 'gunicorn >=19.6.0, <20.0'
+      - 'whitenoise >=3.2.2, <4.0'
+      - 'django-environ >=0.4.1, <1.0'
```

Comment:
Here also better to leave just to the minor version.
Read very good article about this [here](http://semver.org/#semantic-versioning-200)